core/vm/contracts_stateful.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+var (
+	AllowListPrecompileAddress = common.HexToAddress("0x0200000000000000000000000000000000000000")
```

Comment:
Any reason for 0x02....0? I know coreth goes 0x010...n?

Reply:
Wanted to make sure that we do not collide with the precompiles in coreth so that we leave the option open to combine them in the future.